py3status/modules/empty_class.py

```diff
@@ -1,15 +1,35 @@
 # -*- coding: utf-8 -*-
+
+# This is an example module to be used as a template.
+# See https://github.com/ultrabug/py3status/wiki/Write-your-own-modules
+# for more details.
+#
+# NOTE: py3status will NOT execute:
+#     - methods starting with '_'
+#     - methods decorated by @property and @staticmethod
+# 
+# NOTE: reserved method names:
+#     - 'kill' method for py3status exit notification
+#     - 'on_click' method for click events from i3bar (read below please)
+#
+# Do NOT use print on your modules: py3status will catch any output and discard 
+# it silently because this would break your i3bar (see issue #20 for details). 
+# Make sure you catch any output from any external program you may call 
+# from your module. Any output from an external program cannot be caught and 
+# silenced by py3status and will break your i3bar so please, redirect any 
+# stdout/stderr to /dev/null for example (see issue #20 for details). 
+# 
+# Fill in the following docstring: it will beparsed py3status to document
+# your module:
 """
-Empty and basic py3status class.
+One-line summary followed by an empty line.
 
-NOTE: py3status will NOT execute:
-    - methods starting with '_'
-    - methods decorated by @property and @staticmethod
+Multi-line description followed by an empty line.
 
-NOTE: reserved method names:
-    - 'kill' method for py3status exit notification
-    - 'on_click' method for click events from i3bar (read below please)
+@author <your full name> <your email address>
+@license <ancronym>     (please favor BSD or GPLv3)
```

Comment:
I would just replace this and enforce a BSD license just like py3status so packagers don't have to loose time with this from now on